### PR TITLE
Nodes and OSDs in Mirage and controllers.

### PR DESCRIPTION
**Pull request by Ukis**
Sorry for commit "commmit" description. :(

### CODE REVIEW

File: kujira/app/mirage/config.js [hunk 1]
```diff
@@ -1,5 +1,49 @@
+import config from '../config/environment';
+
 export default function() {
 
+this.namespace = config.APP.NAMESPACE;
+
+this.get('/osds', function(db/*, request*/){
+  return {
+    data: db.osds.map(attrs => (
+      { type: 'osd', id: attrs.id, attributes: attrs}
```
Comment (niekrasp) [inline kujira/app/mirage/config.js:10]: When you return list in JSON API, then type should be prural: [JSON API](http://jsonapi.org/format/#document-resource-objects)

File: kujira/app/mirage/config.js [hunk 2]
```diff
@@ -8,7 +52,7 @@ export default function() {
     Note: these only affect routes defined *after* them!
   */
   // this.urlPrefix = '';    // make this `http://localhost:8080`, for example, if your API is on a different server
-  this.namespace = '/kujira/api/v1';    // make this `api`, for example, if your API is namespaced
+  //this.namespace = '/kujira/api/v1';    // make this `api`, for example, if your API is namespaced
```
Comment (niekrasp) [inline kujira/app/mirage/config.js:55]: Why commented out?
It should stay /kujira/api/v1

Comment (Ukis) [inline kujira/app/mirage/config.js:55]: Lines of code before this.namespace are not affected by it.
And second thing is line no. 5: this.namespace = config.APP.NAMESPACE;
So this is actually the same, but better in my opinion (it gets namespace from config).

Comment (niekrasp) [inline kujira/app/mirage/config.js:55]: You are right. I overlooked first occurrence.

**Comment by niekrasp** [comment]
There is no time, files names we will change.

**Pull request merged**